packages/mql-typescript/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@mongodb-js/mql-typescript",
+  "description": "TS definitions for the MongoDB aggregation API",
+  "author": {
+    "name": "MongoDB Inc",
+    "email": "compass@mongodb.com"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://jira.mongodb.org/projects/COMPASS/issues",
+    "email": "compass@mongodb.com"
+  },
+  "homepage": "https://github.com/mongodb-js/devtools-shared",
+  "version": "0.1.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/devtools-shared.git"
+  },
+  "files": [
+    "dist",
+    "bin"
+  ],
+  "bin": {
+    "mql-typescript": "bin/runner.js"
+  },
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "exports": {
```

Comment:
I think we need an export of schema.ts here somewhere so that we can import it from shell-api. Similar to what I started doing [here](https://github.com/mongodb-js/devtools-shared/commit/d7112f2e4894b34a82174461692fc0c167ede656) and what we do in shell api [here](https://github.com/mongodb-js/mongosh/blob/main/packages/shell-api/package.json#L12-L15). 

(Except the real thing so it can be imported, not the giant string.)